src/views/Project/index.jsx

```diff
@@ -113,22 +225,34 @@ export default class Project extends Component {
           'state:open',
         ].join(' ');
 
-        return this.fetch(searchQuery);
+        return this.fetchGithub(searchQuery);
       })
     );
+    const productWithComponentList = mergeAll(
+      project.products
+        ? project.products.filter(product => typeof product !== 'string')
+        : []
+    );
+
+    await Promise.all(
+      Object.entries(productWithComponentList).map(([products, components]) =>
+        this.fetchBugzilla([products], components)
+      )
+    );
```

Comment:
Could you add a comment explaining **why** we are only fetching bugs from products with components but not products without components? This will help people hacking on this in the future to quickly understand.